raiden/ui/web/src/app/components/channel-table/channel-table.component.ts

```diff
@@ -45,22 +44,15 @@ export class ChannelTableComponent implements OnInit, OnDestroy {
             scan((oldChannels, newChannels) => {
                 // use scan and Object.assign to keep object references and
                 // improve *ngFor change detection on data table
-                for (const newchannel of newChannels) {
-                    const oldchannel: WithMenu<Channel> = oldChannels.find((c) =>
-                        c.channel_identifier === newchannel.channel_identifier);
+                for (const newChannel of newChannels) {
+                    const oldChannel: WithMenu<Channel> = oldChannels.find((c) => {
+                        return c.channel_identifier === newChannel.channel_identifier;
```

Comment:
nit picking, no need to make a commit only for that, but arrow functions doesn't need `{ return... }`, you may just put the expression there, and use the full form when complex statements like ifs are required

Reply:
I am aware of this, it was done as a way to break the long line. I mean without having the block there I was not sure where I should break. If you have any suggestions on better handling such cases please let me know :)